install_zowe.sh

```diff
@@ -23,11 +23,15 @@ fi
 # Install the requested version, use the version, and set the default
 # for any further terminals
 
-npm config set @zowe:registry https://zowe.jfrog.io/zowe/api/npm/npm-local-release/
+npm config set @zowe:registry https://zowe.jfrog.io/artifactory/api/npm/npm-local-release/
```

Comment:
Whatever works 😁 Seems like we're still using `https://zowe.jfrog.io/zowe` in the .npmrc for Zowe CLI 🤔